### ISSUE TITLE
Adventures - How to Reach Section - Rome2Rio

### ISSUE THREAD

**Issue by YunusDange** [arshadansari27/fitranginew#141]
This is just a thought , actually we have discussed about this in our Team Meeting we had in your office with everyone.

Rome2Rio is Google Map kind of a thing but it is more than just a google map..it is created for travel websites speciallly...they offer API's to be integrated in any websites...recently they have granted free access to its API to bring multi-modal, visual search to travel sites everywhere. With the new access options, developers will be able to make 100,000 search requests per month without charge.

So it is a great time if we can use and this thing on our Adventure Pages specially in "How to Reach Section" 

It is something altogether different and can give immense mouth publicity to our portal.

check details about Rome2Rio here: 
https://www.rome2rio.com/
https://www.rome2rio.com/documentation/
https://www.rome2rio.com/documentation/whitelabel
http://thenextweb.com/dd/2013/02/07/rome2rio-grants-free-access-to-its-api-to-bring-multi-modal-visual-search-to-travel-sites-everywhere/

this can be done later

**Comment by arshadansari27** [comment]
we had discussed this long time ago.. i was hoping to use this too..

**Comment by YunusDange** [comment]
yup.......just thought about it today & posted it mostly to keep as a reminder  so that we can work over this soon
